### ISSUE TITLE
Items turn invisible in wooden storage crate 

### ISSUE THREAD

**Issue by TheQuadShot** [BluSunrize/ImmersiveEngineering#2065]
This is very strange and I'm very stumped. It only occurs with the crate, no other storage item in my game. I have my HUD set to no fade. I don't use Optifine. After the items turn invisible I can click on an item/s (not a blank/empty slot) the items become visible and I have the item/s I clicked on. If I re-sort the container they also re-appear and then disappear after 10 seconds.

Minecraft: 1.10.2
Forge: 12.18.3.2281
ImmersiveEngineering-0.10-57.jar

F3+C dump file
https://pastebin.com/NHrP1g1E

mod list
https://pastebin.com/TCLHwKwM

**Comment by PapiDimmi** [comment]
x

**Comment by Ash70** [comment]
Having the same problem on 1.12 (had seen it in 1.10 and 1.11 as well).
It does have strange behaviour; I noticed while I was shift-clicking stuff into the crate, at one point everything turned "invisible" but the next stack I shift-clicked into it, went into the first slot (which was previously occupied by something else. Fearing I had lost items, I closed and reopened the crate to see that the original items were still in the first slot and that the 'new' items had moved to another slot.
Sometimes the items take 2 seconds to become 'invisible' and sometimes it can take up to a minute.
It doesn't delete anything, which is good...its just terribly annoying.

**Comment by BluSunrize** [comment]
Can either of you replicate this with just IE installed?

**Comment by TheQuadShot** [comment]
It doesn't in build 10-61.
btw Thanks for making a Great mod!

**Comment by BluSunrize** [comment]
I was going to follow up with saying that it's probably an interaction issue between mods and suggest a binary search...

**Comment by TheQuadShot** [comment]
Sorry, I should have explained. I updated all of the mods in that "pack"(after re-testing in IE only test world) and it didn't repro,

**Comment by Ash70** [comment]
I'll see if I can find the culprit in my pack then. First suspects are malisis doors and storage drawers, since they both glitch as well.

**Comment by mindforger** [comment]
try with inventory tweaks first :)

**Comment by Ash70** [comment]
I don't have inventory tweaks (I use the sort button/restock function from quark).

**Comment by mindforger** [comment]
sorry i mixed up the OP post with yours
well then continue with a binary search if you familiar with that term :)

**Comment by Ash70** [comment]
Ok, did some intensive testing, but did not find the culprit. So in the end I gave up. But when I went back into my regular world, I noticed at some point it happened at the same time when the External Heater texture blinked. So I moved the crate to get a better view, only to find it no longer happened. I then moved all my crates and that solved this issue. Not sure if it has to do with location or if it is a fluke that resets when you move a crate. Anyway; it's solved afaic :-)

On a (un)related note: is the texture blinking on/off from the external heater a known bug?
